src/mapml.css

```diff
@@ -278,12 +279,6 @@
  * User interaction.
  */
 
-/* Disable dragging of controls. */
-.leaflet-control :not([draggable="true"]),
-.mapml-contextmenu :not([draggable="true"]) {
-  -webkit-user-drag: none;
-}
-
```

Comment:
Keeping this shouldn't cause any issues as it doesn't target elements that are `draggable="true"`, and I like it, as it prevents dragging of e.g. zoom controls (as showcased in https://github.com/Maps4HTML/Web-Map-Custom-Element/issues/96), at least in some browsers (there's no support data for `-webkit-user-drag` unfortunately). Maybe controls should just default to `draggable="false"` instead?

Reply:
Alright, I'll re-add that snippet, do you know why the cursor isn't the grab cursor? There's a drag-active class I added that should set the cursor to 'grabbing' but it doesn't seem to work, it keeps the 'not-allowed' cursor. If you have any suggestions let me know.

Reply:
It seems that `preventDefault()` is needed on the `dragover` event to allow drop, the intended behavior can be seen in [this example](https://developer.mozilla.org/en-US/docs/Web/API/Document/dragover_event#examples).

btw, since we're pulling in leaflet.css we could save a few lines of code by utilizing [`leaflet-grab`](https://github.com/Leaflet/Leaflet/blob/436430db4203a350601e002c8de6a41fae15a4bf/dist/leaflet.css#L211-L215) and not duplicate that CSS in a new `drag-active`. 👍🏼

Reply:
I think I'll still need the `drag-active` as it also changes the opacity of what's being dragged, this makes clear where the dragged element is being moved, but I can get rid of the cursor related CSS in `drag-active` as it is duplicated. Thanks

Reply:
I just realized Leaflet sets `leaflet-dragging` on the document `<body>`,  so the selector `.leaflet-dragging .leaflet-grab` wont work because that's outside the shadow boundary... I suppose we might as well have a custom class for this then, alternatively set `leaflet-dragging` on some parent element of `leaflet-grab`.

Reply:
@ahmadayubi I have a fix to allow for using `class="leaflet-grab"` in https://github.com/Malvoz/Web-Map-Custom-Element/commit/c47db9d7a4016607373935132ad2c5e1ea22900b, do you want to incorporate that in this PR, or should I create a separate PR for that? **We'll need that regardless**, because the issue described in https://github.com/Maps4HTML/Web-Map-Custom-Element/pull/249#discussion_r561320063 means that dragging the map display currently doesn't show the `grabbing` cursor (only `grab`).

Reply:
Maybe keep it a separate PR or delay adding it to this one, I'm currently working on using a custom dragging implementation to limit the axis the user can drag on, so the control item can only be moved up and down within the controls overlay rather than anywhere the mouse goes.

Reply:
Okay I opened a PR, but please ignore my last https://github.com/Maps4HTML/Web-Map-Custom-Element/pull/249#discussion_r561346566, although the PR fixes `cursor: grabbing` for dragging the map display, it relies on `<mapml-viewer class="leaflet-drag-target">`. So going back to using a custom `drag-active` class is probably the easiest way, even if it means duplicating CSS.